lib/MiLight/MiLightClient.cpp

```diff
@@ -2,40 +2,42 @@
 #include <MiLightRadioConfig.h>
 #include <Arduino.h>
 
-MiLightRadio* MiLightClient::switchRadio(const MiLightRadioType type) {
+
+MiLightRadioInterface* MiLightClient::switchRadio(const MiLightRadioType type) {
   RadioStack* stack = NULL;
-  
+
   for (int i = 0; i < numRadios; i++) {
     if (radios[i]->config.type == type) {
       stack = radios[i];
       break;
     }
   }
-  
+
   if (stack != NULL) {
-    MiLightRadio *radio = stack->getRadio();
-    
+    MiLightRadioInterface *radio = stack->getRadioInterface();
+
     if (currentRadio->config.type != stack->config.type) {
       radio->configure();
     }
-    
+
     currentRadio = stack;
     formatter = stack->config.packetFormatter;
     return radio;
   } else {
-    Serial.print(F("MiLightClient - tried to get radio for unknown type: "));
+    Serial.print("MiLightClient - tried to get radio for unknown type: ");
```

Comment:
It should - I think I started the code before you made the PROGMEM macro addition. That is why it's different.